src/main/java/org/mule/extension/sftp/internal/connection/provider/SftpConnectionProvider.java

```diff
@@ -153,10 +152,12 @@ public SftpFileSystemConnection connect() throws ConnectionException {
         } else if (e.getMessage().contains("Connection reset by peer")) {
           throw new SftpConnectionException(getErrorMessage(connectionSettings, e.getMessage()), e, FileError.CONNECTIVITY);
         }
-      }
-      if (e.getDisconnectCode() == 9) {
+      } else if (e.getDisconnectCode() == 9) {
         throw new SftpConnectionException(getErrorMessage(connectionSettings, e.getMessage()), e, FileError.CANNOT_REACH);
+      } else {
+        throw new SftpConnectionException(e.getMessage(), e, FileError.CONNECTIVITY);
```

Comment:
Do we want any SshException that hasn't been catched to be CONNECTIVITY?

Reply:
This was a test I will remove it